components/function-controller/pkg/apis/serverless/v1alpha2/function_defaults_test.go

```diff
@@ -265,9 +265,10 @@ func TestSetDefaults(t *testing.T) {
 			},
 			expectedFunc: Function{
 				Spec: FunctionSpec{
-					ResourceConfiguration: ResourceConfiguration{
-						Function: ResourceRequirements{
-							Resources: corev1.ResourceRequirements{
+					Runtime: NodeJs14,
```

Comment:
Runtime is not set by default anymore: https://github.com/kyma-project/kyma/issues/15289